src/benchmarks/micro/coreclr/perflab/CastingPerf.cs

```diff
@@ -373,5 +374,66 @@ public bool CheckArrayIsInterfaceNo()
                 res = myClass2Arr is IMyInterface1[];
             return res;
         }
+
+        [GlobalSetup(Target = nameof(CheckArrayIsNonvariantGenericInterface))]
```

Comment:
There's a fair bit of repetition with these setup methods. BDN has a different overload of `GlobalSetup` which takes an array:
 ```suggestion
        [GlobalSetup(Targets = new[] {  nameof(CheckArrayIsNonvariantGenericInterface), nameof(CheckArrayIsNonvariantGenericInterfaceNo)})]
```

Reply:
I did not know such overload exists. Will change of course. 
It was a bit annoying to repeat the same thing. Thanks!